game/external/discord_jak2.cpp

```diff
@@ -0,0 +1,84 @@
+#include "discord_jak2.h"
+
+namespace jak2 {
+const std::map<std::string, std::string> level_names = {
```

Comment:
there are also these ones I don't see being grouped into a larger level or otherwise handled:
- `sagehut` (samos hut in dead town)
- `atoll-ext`
- `mtnext`
- `forestb`
- `drillb`
- `drillmtn`
- `dig3b`
- `consiteb`

Reply:
From what I could tell, most of these won't ever be returned by `level-get-target-inside`, so I didn't bother with them, but I could still add them as failsafes.

Reply:
`consiteb` is definitely returned, it's the airlock level.